components/centraldashboard/public/components/main-page.pug

```diff
@@ -12,8 +12,11 @@ app-drawer-layout.flex(narrow='{{narrowMode}}',
             a(href$='[[_buildHref("/", queryParams)]]', tabindex='-1')
                 paper-item.menu-item Home
                 aside.divider
+            a(href  ='https://www.kubeflow.org/docs/about/kubeflow/',
+                tabindex='-1', target="_new")
+                paper-item.menu-item Documentation
```

Comment:
Can we add an external link looking icon on this? The way pipelines used to do this.

Perhaps:
![image](https://user-images.githubusercontent.com/5303018/56764166-f4999680-6758-11e9-853f-849bfa148954.png)

Reply:
Not sure how those extra spaces got there. I went with the launch icon since that's what we used for the other doc links on the landing page. I think we should probably try to be consistent but I'm open to your thoughts as well.

Reply:
Yeah, I was looking for it, forgot we already had that icon in our app. This is good!
Mind if I make a quick change in your PR?